remix-tests/README.md

```diff
@@ -1,51 +1,28 @@
-Remix-Tests
----
+## Remix-Tests
+[![npm version](https://badge.fury.io/js/remix-tests.svg)](https://www.npmjs.com/package/remix-tests)
+[![npm](https://img.shields.io/npm/dt/remix-tests.svg?label=Total%20Downloads)](https://www.npmjs.com/package/remix-tests)
+[![npm](https://img.shields.io/npm/dw/remix-tests.svg)](https://www.npmjs.com/package/remix-tests)
+[![GitHub](https://img.shields.io/github/license/mashape/apistatus.svg)](https://github.com/ethereum/remix/tree/master/remix-tests)
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/ethereum/remix/issues)
 
-> Tests for the Ethereum tool suite [Remix](https://github.com/ethereum/remix)
 
-### Installation
-
-`npm -g install remix-tests`
-
-### Test structure
-
-Example test file:
-```Javascript
-pragma solidity ^0.4.7;
-import "remix_tests.sol"; // injected by remix-tests
-import "./simple_storage.sol";
+`remix-tests` is a tool to test Solidity smart contracts. It works underneath Remix IDE plugin "Solidity Unit Testing" which is used to write and run test cases for a contract. Tests are written in Solidity itself. `remix-tests` can be used as CLI and a library too.
 
-contract MyTest {
-  SimpleStorage foo;
-  uint i = 0;
-
-  function beforeAll() {
-    foo = new SimpleStorage();
-  }
+### Installation
+As a dependency inside file:
 
-  function beforeEach() {
-    if (i == 1) {
-      foo.set(200);
-    }
-    i += 1;
-  }
+`npm install --save remix-tests`
 
-  function initialValueShouldBe100() public {
-    Assert.equal(foo.get(), 100, "initial value is not correct");
-  }
+As a dev dependency:
 
-  function initialValueShouldBe200() public constant returns {
-    return Assert.equal(foo.get(), 200, "initial value is not correct");
-  }
+`npm install --save-dev remix-tests`
 
-}
-```
+As a global NPM module to use as CLI:
 
-See also: example [Su Squares contract](https://github.com/su-squares/ethereum-contract/tree/e542f37d4f8f6c7b07d90a6554424268384a4186) and [https://travis-ci.org/su-squares/ethereum-contract/builds/446186067](Travis build) that uses remix-tests for continuous integration testing.
+`npm -g install remix-tests`
```

Comment:
I don't really see the point of making it global...

Reply:
If it's in `devDependencies` they can do `npx remix-tests ....`

Reply:
This can be used in case where one want to run just a test file with a node project